kolibri/utils/cli.py

```diff
@@ -273,3 +274,8 @@ def main(args=None):
         plugin_name = arguments['PLUGIN']
         plugin(plugin_name, **arguments)
         return
+
+    if arguments['start']:
+        from . import server
```

Comment:
Local import here was to eliminate a circular import with `from kolibri.deployment.default.wsgi import application`, which itself bootstraps the django environment. Could probably flip the local-import and have server.py do the wsgi import inside "run_server"?